bundler/cli.js

```diff
@@ -24,19 +25,27 @@ function add(value) {
 	command = null
 }
 
+function format(n) {
+	return n.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1,")
+}
+
 bundle(params.input, params.output, {watch: params.watch})
 if (params.minify) {
-	minify(params.output, params.output, {watch: params.watch, advanced: params.aggressive}, function(stats) {
-		function format(n) {
-			return n.toString().replace(/(\d)(?=(\d\d\d)+(?!\d))/g, "$1,")
-		}
-
-		console.log("Original size: " + format(stats.originalGzipSize) + " bytes gzipped (" + format(stats.originalSize) + " bytes uncompressed)")
-		console.log("Compiled size: " + format(stats.compressedGzipSize) + " bytes gzipped (" + format(stats.compressedSize) + " bytes uncompressed)")
+	// mFiles = { original: String(mithril.js), compressed: String(mithril.min.js) }
+	var mFiles = minify(params.output, params.output, {watch: params.watch, advanced: params.aggressive})
```

Comment:
`advanced` seems obsolete

Reply:
also `params.output, params.output,`?? It would overwrite the file right?

Reply:
`params.input` is mithril.js. `params.output` is the just-built mithril.min.js, which isn't yet minified. So yes, this call is for an overwrite.

Reply:
not sure if `advanced` (`params.aggressive`) is being used elsewhere or not — just didn't want to break anything.

Reply:
If it's a no-op, it's safe to remove. And UglifyJS doesn't have any "advanced" option - they only have two levels of minification: either minify or don't. So go ahead and drop `advanced: params.aggressive`.

Reply:
@StephanHoyer @isiahmeadows ok, `advanced/aggressive` are gone.

The remaining question: since `bundler/cli.js` is the only place where we use `bundler/minify.js`, I'm ditching `inputPath/outputPath` for `filePath`.